plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/router/ERDConnectionRouterDescriptor.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.erd.ui.router;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.eclipse.draw2d.AbstractRouter;
+import org.eclipse.draw2d.PolylineConnection;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.model.impl.AbstractDescriptor;
+import org.jkiss.dbeaver.registry.RegistryConstants;
+import org.jkiss.utils.CommonUtils;
+
+public class ERDConnectionRouterDescriptor extends AbstractDescriptor {
+    private String id;
+    private String name;
+    private String description;
+    private boolean isDefault = false;
+    private boolean isEnableAttributeAssociation;
+    private AbstractRouter router;
+    private ObjectType lazyRouter;
```

Comment:
LEt's add function `createConnection` into ERDConnectionRouter instead of mapping 2 attributes to class contribution.

Reply:
The connection class packed to router, in order to provide clear logic.